transformer.md

```diff
@@ -255,3 +255,62 @@ The end marker is the following, where _`<N>`_ is the number of the step:
 ```markdown
 <!-- Killercoda step<N>.md END -->
 ```
+
+## Generate a tutorial
+
+You can generate a new tutorial from existing documentation using the transformer tool.
+After generation, a [GitHub Actions workflow](./.github/workflows/regenerate-tutorials) updates the tutorial when the documentation source changes.
+
+### Before you begin
+
+1. Clone the repository with the source documentation.
+1. Clone the Killercoda repository.
+1. [Download and install Go](https://go.dev/doc/install) to run the transformer tool.
+
+To generate a tutorial:
+
+1. In the source repository, check out a new branch.
+
+   Give the branch a name that reflects the planned changes.
+   For example, `2024-06-killercoda-tutorial-for-loki-quickstart`.
+
+1. In the Killercoda repository, check out a new branch.
+
+   For convenience, name the branch the same as you did in the source repository.
+
+1. In the source repository, run `make docs` from the `docs/` directory so that you can verify your changes don't break the rendered documentation.
+
+1. In the source file, add the Killercoda front matter.
```

Comment:
Perhaps we could add a link here to what front matter is. I always thought of it as page metadata so took me a second to work out what front matter was

Reply:
Good point! Front matter is somewhat specific to Hugo so I'll link to something that explains that.